Makefile

```diff
@@ -89,6 +89,10 @@ hermetic-tests:
 		--clean-up \
 		--test-image ${TEST_SUITE_IMAGE}
 
+.PHONY: run-suite
```

Comment:
This is for durable tests correct? Should we rename this to `durable-tests`? Also how do you choose the environment to target? Should we a make var called DURABLE_ENV or something?

Reply:
Yeah, not very different from hermetic, except that they can be run wherever there's Docker running. Kind of like a `docker-compose` variant to the hermetic tests, which will only run in k8s.

I'll change the script to run only against durable infrastructure.